.github/workflows/build-waypoint-oss.yml

```diff
@@ -0,0 +1,117 @@
+---
+name: build_waypoint
+
+on:
+  workflow_call:
+    inputs:
+      bundle-path:
+        required: false
+        type: string
+      cgo-enabled:
+        type: string
+        default: 0
+      create-packages:
+        type: boolean
+        default: true
+      goos:
+        required: true
+        type: string
+      goarch:
+        required: true
+        type: string
+      go-tags:
+        type: string
+      go-version:
+        type: string
+      package-name:
+        type: string
+        required: true
+      waypoint-version:
+        type: string
+        required: true
+      waypoint-metadata:
+        type: string
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      bin-name: ${{ steps.get-bin-name.outputs.bin-name }}
+    name: Build ${{ inputs.package-name }} ${{ inputs.goos }} ${{ inputs.goarch }} v${{ inputs.waypoint-version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Restore ui assets
+        uses: actions/cache/restore@v3
+        with:
+          path: ui/dist/assets
+          key: ui_assets_${{ github.sha }}
+          fail-on-cache-miss: true
+          enableCrossOsArchive: true
+      - name: Restore build assets
+        uses: actions/cache/restore@v3
+        with:
+          path: internal/assets
+          key: build_assets_${{ github.sha }}
+          fail-on-cache-miss: true
+          enableCrossOsArchive: true
+      - name: Determine binary name
+        id: get-bin-name
+        run: |
+          bin_name="${{ inputs.package-name }}"
+          if [ "${{ inputs.goos }}" == "windows" ];then
+               bin_name="${bin_name}.exe"
+          fi
+          echo "bin-name=${bin_name}" | tee -a "${GITHUB_OUTPUT}"
+      - name: Build binary
+        env:
+          GOOS: ${{ inputs.goos }}
+          GOARCH: ${{ inputs.goarch }}
+          WAYPOINT_GOOS: ${{ inputs.goos }}
+          WAYPOINT_GOARCH: ${{ inputs.goarch }}
+          GO_LDFLAGS: ${{ inputs.ld-flags }}
+          ACTIONSOS: ${{ inputs.runson }}
+          CGO_ENABLED: ${{ inputs.cgo-enabled }}
+          CRT_BIN_NAME: ${{ steps.get-bin-name.outputs.bin-name }}
+        uses: hashicorp/actions-go-build@v0.1.7
+        with:
+          product_name: ${{ inputs.package-name }}
+          product_version: ${{ inputs.waypoint-version }}
+          go_version: ${{ inputs.go-version }}
+          os: ${{ inputs.goos }}
+          arch: ${{ inputs.goarch }}
+          reproducible: report
+          instructions: |
+            make bin/crt-${{ inputs.package-name }}
+        # FIXME: are we still only publishing linux amd64 packages?
```

Comment:
@waypoint - Please confirm that the only arch we are supporting for linux packaging is `amd64`?

Reply:
Update to `amd64` and `arm` for linux packages